src/configuration/useCopy.ts

```diff
@@ -0,0 +1,15 @@
+import { useTranslation } from "react-i18next"
+
+import copy from "./copy.json"
+
+type CopyKey = "welcome_message" | "about" | "legal"
+
+const useCopy = (key: CopyKey): string => {
```

Comment:
What do you think about a more specific name?

```suggestion
const useAuthorityCopy = (key: CopyKey): string => {
```